### ISSUE TITLE
Twitter?

### ISSUE THREAD

**Issue by sergiolepore** [banodejs/Bienvenido#3]
Hola! La cuenta de Twitter que figura en el readme ([@banode](https://twitter.com/banode)), es correcta? Es una cuenta muy sospechosa a nombre de "monika banode". Está de placeholder, verdad?

**Comment by a0viedo** [comment]
Nono, en realidad existe una Monika con apellido Banode (desafortunadamente). Por el momento no existe cuenta de twitter oficial. Tal vez es un buen tema para proponer.

**Comment by nescalante** [comment]
A mi me gusta este: [@nodejsba](https://twitter.com/nodejsba)

**Comment by a0viedo** [comment]
dejo otros dos:
- [@nodebsas](https://twitter.com/nodebsas)
- [@bsasnode](https://twitter.com/bsasnode)

**Comment by alfathenus** [comment]
Propongo otro mas @banodejs

**Comment by a0viedo** [comment]
cc @banode/miembros @banode/owners

**Comment by bevacqua** [comment]
+1 a @banodejs, pero renombraria la org para q matchee con la cuenta en twitter

**Comment by farolfo** [comment]
@banodejs o asesinar a monika banode jaja

**Comment by eafelix** [comment]
jajajajajjaja banodejs me gusta, banode tambien, que dilema

**Comment by bevacqua** [comment]
Qué lástima que @buenode está tomado, era bueno..de (?)

**Comment by ipeluffo** [comment]
+1 a @banodejs ... es una desventaja que la organización quede con otro nombre pero los organizadores/owners sabrán si conviene o se puede renombrar

**Comment by nescalante** [comment]
+1 a @banodejs

- Conviene renombrar
- Se puede renombrar

**Comment by bevacqua** [comment]
https://twitter.com/banodejs

**Comment by vrunoa** [comment]
:+1:

**Comment by a0viedo** [comment]
Falta:
- [ ] Cambiar todas las menciones de BANode a BANodejs (o BANode.js?)
- [ ] Crear los gitter rooms
- [ ] Actualizar las gitter badges

**Comment by bevacqua** [comment]
Agreed. Pero dejaría las menciones tal y como están (mi voto secundario es para BANode.js), ya que BANode.js dificilmente la gente lo incorpore

**Comment by a0viedo** [comment]
Entonces todos están de acuerdo que el sitio sea banodejs.org?

**Comment by bevacqua** [comment]
No dan `.org.ar`?

**Comment by a0viedo** [comment]
@bevacqua ponele que si, aún así prefiero `.org` antes que `.org.ar`...

**Comment by bevacqua** [comment]
Cierto, no da `nodejs.org.ar` (copyright issues)

**Comment by sergiolepore** [comment]
+1 banodejs en todos lados :)

**Comment by alfathenus** [comment]
me gusta banodejs.org  el .org.ar es solo para el estado creo....

**Comment by jkutianski** [comment]
El dominio “org.ar” es para cualquier organización sin fines de lucro con personería jurídica.


https://nic.ar/normativa-vigente.xhtml



.ORG.AR

Sólo podrán registrar dominios en la zona .org.ar las entidades que sean organizaciones sin fines de lucro argentinas o extranjeras. No podrán registrar nombres en esta zona las personas físicas por más que la actividad que las mismas desempeñen carezca de fines de lucro. La documentación a presentar debe respaldar este carácter:
Nota de Validación de Persona Jurídica (DPJ), tildando la opción correspondiente a la zona: https://nic.ar/documentacion.xhtml#DPJ
Acta constitutiva o estatuto, de donde surja el carácter de organización sin fines de lucro
Acto de nombramiento del funcionario firmante
DNI del firmante






De: Andres Lozada Mosto
Enviado el: ‎viernes‎, ‎27‎ de ‎febrero‎ de ‎2015 ‎12‎:‎57‎ ‎p.m.
Para: banodejs/Bienvenido





me gusta banodejs.org el .org.ar es solo para el estado creo....
